src/Plugin.cc

```diff
@@ -42,6 +43,18 @@ Plugin::Plugin()
 {
 }
 
+/////////////////////////////////////////////////
+Plugin::Plugin(const std::string &_filename, const std::string &_name,
+               const std::string &_xmlContent)
+  : dataPtr(std::make_unique<sdf::PluginPrivate>())
+{
+  this->SetFilename(_filename);
+  this->SetName(_name);
+  std::string trimmed = ignition::common::trimmed(_xmlContent);
```

Comment:
you are only using ignition::common here, can you remove this dependency here ?

Reply:
Done in [e6fc820](https://github.com/ignitionrobotics/sdformat/pull/911/commits/e6fc820d074fc8f07a654210142e147a69c909d9)